### PR TITLE
[UPDATE BONUS GUIDE] Update system-overview.md to reference raspibolt-pulse

### PR DESCRIPTION
**Pull request by vicariousdrama**
- References raspibolt-pulse repo in place of system-overview sample.
- assumes eth0 over wlan0, inverts guidance
- changed guidance to review script content before installing

#### What

Migrate from older system-overview to raspibolt-pulse

### Why

References the scripts in new raspibolt-pulse repo which are intended to work across versions of Raspibolt (1.x - 3.x), and provide info about alternative electrum server, lighting implementation, and lightning web app

#### How

References new repo and updated commands to install

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [ ] simple bug fix

Fixes # (link issue)

Depends on https://github.com/raspibolt/raspibolt-pulse/pull/17

### CODE REVIEW

**Review by M1ch43lV — APPROVED** [review]

**Comment by zynos** [comment]
Can we merge this now?
@vicariousdrama Updated his pull request already

**Comment by vicariousdrama** [comment]
Yes. This was dependent on the raspibolt-pulse PR which has been merged.

File: guide/bonus/raspberry-pi/system-overview.md [hunk 1]
```diff
@@ -33,39 +33,27 @@ This script can be run by user "admin" without root privileges, but you should s
...
   ```
 
-* Inspect the script to make sure it does not do bad things.
+* Inspect the scripts to make sure it does not do bad things.
   Exit with `Ctrl`-`X`
 
   ```sh
-  $ nano 20-raspibolt-welcome --linenumbers
+  $ cd /tmp/raspibolt-pulse
+  $ nano *.sh --linenumbers
```
Comment (blckbx) [inline guide/bonus/raspberry-pi/system-overview.md:48]: Guess, we don't need to edit now?

**Review by blckbx — COMMENTED** [review]
One nit otherwise LGTM.

File: guide/bonus/raspberry-pi/system-overview.md (hunk 1 above)
Comment (vicariousdrama) [reply guide/bonus/raspberry-pi/system-overview.md:48]: No editing is required. Should we drop the recommendation to review it?

Comment (blckbx) [reply guide/bonus/raspberry-pi/system-overview.md:48]: Ah yes, for verification. Missed the comment from above.

**Review by blckbx — APPROVED** [review]
LGTM

**Pull request merged**